src/com/google/javascript/jscomp/PolymerClassDefinition.java

```diff
@@ -33,6 +33,8 @@
  * class.
  */
 final class PolymerClassDefinition {
+  /** Either 1 or 2 - the Polymer version of the definition */
+  final int version;
```

Comment:
Polymer()-created elements do not need to be hybrid - they could use Polymer 2.0 only features, or Web Components v1 features. They are "legacy" in that they have another layer of 1.x back-compat APIs added to them.

Ack on the difference between renaming ability, but also, there's a chance we'll introduce a new class-based syntax for 1.x for better forward compatibility with 2.x, in which case Polymer() vs class, and renaming vs. not might be completely orthogonal.